airflow/providers/opensearch/hooks/opensearch.py

```diff
@@ -40,15 +46,49 @@ class OpenSearchHook(BaseHook):
     conn_type = "opensearch"
     hook_name = "OpenSearch Hook"
 
-    def __init__(self, open_search_conn_id: str, log_query: bool, **kwargs: Any):
+    def __init__(
+        self,
+        open_search_conn_id: str,
+        log_query: bool,
+        open_search_conn_class: str = "RequestsHttpConnection",
```

Comment:
I don't think we need to use such complicated logic with dynamic imports, we could just provide class as is into the hook/operator

```python
from typing import Any, TYPE_CHECKING

if TYPE_CHECKING:
    from opensearchpy import Connection as OpenSearchConnectionClass

class OpenSearchHook(BaseHook):
    def __init__(
        ...
        open_search_conn_class: type[OpenSearchConnectionClass] | None = None,
```

Reply:
And also need add information about new attribute in docstrings

Reply:
I think I have done the requested changes, could you take another look?